### PR TITLE
Feature: add exprtk calc.

### PR DESCRIPTION
**Pull request by poetaster**
Implement programming with exprtk.hh affording inputs: 
```
var a:=0; var b:=2; var v[3]:={5,10,15};
for ( a ; a < v[] ; a += 1) { b+=v[a]; 
```
Still missing docs and variable extraction/saving.

### CODE REVIEW

**Comment by poetaster** [comment]
The last commit fixes variable and result lists so that the caculations don't pollute other views with verbose vars/formulas. The Variables and results can be used on other views.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+name: Build RPMs
```
Comment (rabauke) [inline .github/workflows/build.yaml:1]: Can you please move the build automation script into a separate pull request?  This is a useful contribution, though I do not see any direct relation to the other changes.

**Review by rabauke — CHANGES_REQUESTED** [review]
More detailed review will follow.

File: .github/workflows/build.yaml (hunk 1 above)
Comment (poetaster) [reply .github/workflows/build.yaml:1]: sorry, it's the consequence of working on master. sigh. I'll see if I can remove it from this pr. I should have squashed, too, but I got a bit carried away.

Comment (poetaster) [reply .github/workflows/build.yaml:1]: hmmm. rebasing seems premature. perhaps  best is if you look at this and when you've had time I'll restructure the PR into multiple ones. I'm also working on docs/interactive help/completion.

Comment (poetaster) [reply .github/workflows/build.yaml]: > Can you please move the build automation script into a separate pull request? This is a useful contribution, though I do not see any direct relation to the other changes.

Ah, I reverted it  and ```git push origin HEAD:master --force``` sufficed.

**Comment by poetaster** [comment]
A couple of notes:
1. my initial inspiration for using exprtk comes form: https://github.com/vetux/qcalculator which uses mpdecimal for faster more accurate decimal math. That complicates building and I thought I'd first try creating librarys for sfos that provide mpdecimal and exprtk 
2. qcalculator implments interfaces to symboltable (vars, constants, formulas, scripts) which I intend to implement). 
3. storage of symboltable data, since it can contain complex expressions and vectors, I had intended using localstorage, sqllite for

**Comment by poetaster** [comment]
I have a branch for Documentation related work . I've implemented a small change in the Scientific and Programmable views as can be seen here:
https://github.com/poetaster/harbour-babbage/issues/1

Also implemented is a copy mechanism from the operator functions list which, in the case of programmable, includes the ability to copy examples which are more complex.

I noted that loading the Functions.qml was quite slow, so I implemented a javascript aproach (json docment) which seems faster. The downside of that is that it does not conform to the standard methods of localization.

**Comment by poetaster** [comment]
I've done significant work on other branches. I'm going to close this request and merge the relevant changes, cherry pick some into separate PRs for you. It would like something like:
1. PR github action
2. PR inline help in Scientific view (with copy of functions from Functions.qml)
3. PR exprtk inclusion.
